atomicapp/providers/lib/kubeconfig.py

```diff
@@ -103,16 +106,52 @@ def parse_kubeconf_data(kubecfg):
         logger.debug("user: %s", user)
 
         url = cluster["cluster"]["server"]
-        token = user["user"]["token"]
-        if "namespace" in context["context"]:
-            namespace = context["context"]["namespace"]
-        if "insecure-skip-tls-verify" in cluster["cluster"]:
-            tls_verify = not cluster["cluster"]["insecure-skip-tls-verify"]
-        elif "certificate-authority" in cluster["cluster"]:
-            ca = cluster["cluster"]["certificate-authority"]
+        auth = user["user"].get("token")
+        namespace = context["context"].get("namespace")
+        tls_verify = not cluster["cluster"].get("insecure-skip-tls-verify")
+
+        if tls_verify:
+            ca_data = cluster["cluster"].get("certificate-authority-data")
+            if ca_data:
+                ca = Utils.getTmpFile(b64decode(ca_data))
+            else:
+                if "certificate-authority" in cluster["cluster"]:
+                    # if we are in container translate path to path on host
+                    ca = os.path.join(Utils.getRoot(),
+                                      cluster["cluster"].get("certificate-authority").lstrip('/'))
```

Comment:
really long line.. can we break it up into two? maybe `cluster["cluster"].get("certificate-authority").lstrip('/')` on the first and the `join` on the 2nd.